tests/vic-machine.bats

```diff
@@ -123,6 +124,12 @@ teardown() {
     run "$docker" "${docker_cmd[@]}" start "$name"
     assert_success
 
+    run wget "$vicadmin_ip"/container-logs.tar.gz --no-check-certificate -O install-container-logs.tar.gz
```

Comment:
Hmm, you're right. I'll have to break this test into two tests, then, since the first VM is destroyed in the middle of the test. Really those vm destruction steps should be in teardown too..